build/postgres/Dockerfile

```diff
@@ -51,6 +89,7 @@ else \
 		psmisc \
 		rsync \
 		wal2json${PG_MAJOR//.} \
+		file \
```

Comment:
Ensure that postgres images has necessary packages to run in all modes when `DFSET` is centos or ubi/rhel